.github/workflows/scheduled.yaml

```diff
@@ -21,6 +21,9 @@ jobs:
       strategy: ${{ matrix.test-strategy }}
       additional-env-vars: |
         PKG_SYSREQS_DRY_RUN=true
+      extra-deps: |
+        dplyr (>= 1.0.9),
```

Comment:
This seems to be redundant. It is a direct dependency and in the DESC file we have exactly the same version.

Reply:
yes, agreed. there is quite a few changes around this I think. we will come back to this slowly